python/doc/examples/surrogate_modeling/gaussian_process_regression/plot_gpr_1d.py

```
@@ -78,30 +79,60 @@
 # In order to observe the function and the location of the points in the input design of experiments, we define the following function which plots the data.
 
 
-def plot_1d_data(x_data, y_data, type="Curve", legend=None, color=None, linestyle=None):
-    """Plot the data (x_data,y_data) as a Cloud/Curve"""
+def plot_1d_data(
+    x_data,
+    y_data,
+    type="Curve",
```

Ruff (0.15.14)

[error] Function argument `type` is shadowing a Python builtin

(A002)